### PR TITLE
Adds FML to up-for-grabs.net

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: _data/projects/fml.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: FML
+desc: A CLI tool that tells a joke when you type fml in the terminal
+site: https://github.com/slash-arun/fml
+tags:
+- python
+- cli
+upforgrabs:
+  name: help wanted
+  link: https://github.com/slash-arun/fml/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
Comment (shiftkey) [inline _data/projects/fml.yml]:
```suggestion
  link: https://github.com/slash-arun/fml/labels/help%20wanted
```

**Review by shiftkey — CHANGES_REQUESTED** [review]
Just one suggestion to tidy things up:

**Comment by ghost** [comment]
@shiftkey Addressed the comment.

**Review by shiftkey — APPROVED** [review]
Thanks @slash-arun!

**Pull request merged**